app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="container flex flex-col justify-between h-screen bg-slate-100 px-0 mx-auto">
+  <div class="flex-1 m-0">
+    <div class="flex justify-between items-center bg-blue-600 py-4 px-3">
+      <%= link_to groups_path do %>
+      <span class="text-white" style="font-size:24px;">&#10229;</span>
+      <% end %>
+      <span class="uppercase text-white font-medium">TRANSACTIONS</span>
+      <%= button_to "", class: "bg-blue focus:border-white" do %>
+      <i class="fas fa-search text-white p-2" style="font-size:24px;"></i>
+      <% end %>
+    </div>
+    <div class="list-group-item shadow-sm justify-between items-center mb-3">
+      <div class="flex justify-start px-3 gap-2">
+        <%= image_tag url_for(@group.icon),
+        alt: @group.name,
+        class: "object-fit-fill rounded-0 float-left h-16 w-16" %>
+        <div class="flex flex-1 justify-between w-full py-4 px-1">
+          <span class="items-center font-medium"><%= @group.name %></span>
+          <span class="items-center font-medium"><%= number_to_currency(@group.total_amount) %></span>
+        </div>
+      </div>
+    </div>
+    <ul class="list-group rounded-0 px-3 mt-3">
+      <% @expenses.each do |expense| %>
+      <li class="list-group-item bg-white drop-shadow-lg justify-between items-center mb-3">
+        <div class="flex justify-between items-center bg-white w-full p-4 gap-4">
+          <span class="flex-1 text-left"><%= expense.name %></span>
+          <span class="text-right"><%= number_to_currency(expense.amount) %></span>
+        </div>
+      </li>
+      <% end %>
+    </ul>
+  </div>
+  <div class="w-full flex bg-green-500 text-white rounded-0 m-0 p-4">
+    <%= link_to "Add New Transaction",
+    new_group_expense_path(@group), class: "bg-green-500 font-medium text-center border border-white flex-1 text-white rounded-0  py-2 m-0 hover:bg-green-700 active:bg-green-800" %>
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done but on my side I can not add new `Transactions` to other `Categories` and `Save(Add)` button is not working during the `Mobile` format. Please fix them 🙏🏼 

![image](https://github.com/Ridwanullahi-code/BudgetApp/assets/63915024/5d352864-9b98-4540-bb50-4c978b97acfb)

![image](https://github.com/Ridwanullahi-code/BudgetApp/assets/63915024/b4ded6fd-59d1-492f-845e-a9441c29c7b9)